openfe/analysis/plotting.py

```diff
@@ -5,7 +5,7 @@
 from matplotlib.axes import Axes
 import numpy as np
 import numpy.typing as npt
-from openff.units import unit
+from openff.units import unit, Quantity
```

Comment:
It shouldn't make a difference in import times in this case, but if shifting from `unit.Quantity` to `Quantity`, the `unit` namespace shouldn't be needed